package.json

```diff
@@ -18,7 +18,7 @@
   "scripts": {
     "clean": "rimraf output && rimraf .pulp-cache",
     "docs": "pulp docs",
-    "build": "pulp build -- --censor-lib --strict",
```

Comment:
While it's true there are no warnings currently, it's probably better to keep it, in case they arise again in the future.